demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -62,6 +62,16 @@ class _Argument(BaseStrictModel):
 )
 
 
+class BaseVersionYaml(BaseStrictModel):
```

Comment:
please use in all applicable items

Reply:
Note: This only applies if those two fields are not required

Reply:
then either add a comment in code, or rename the class (and its jsonic twin) `BaseOptionalVersionYaml`

Reply:
Done